### PR TITLE
Add simple toast - simple markup and CSS

### PR DESCRIPTION
**Pull request by knagode**
I simply copy/pasted CSS from Polaris. Not sure if this is the best strategy. 
I added some dummy JS to make it work in demo (add/remove show class on toast object).

Demo: https://knagode.github.io/bootstrap-polaris/demo/#toast
(is there a way to copy/past demo dir to github.io pages automatically? - I used copy/paste strategy :) )

### CODE REVIEW

File: demo/index.html [hunk 1]
```diff
@@ -793,6 +793,28 @@ <h3>USPS has updated their rates</h3>
...
+          <div class="card-body">
+            <h3>Basic Toast</h3>
+            <button id='toastButton' type="button" class="btn" onclick="var show = this.innerHTML == 'Show Toast'; var obj = document.getElementById('sampleToast'); if(show) { obj.classList.add('show'); this.innerHTML = 'Hide Toast'; } else { obj.classList.remove('show'); this.innerHTML = 'Show Toast'; }">Show Toast</button>
+          </div>
+        </div>
+      </div>
+    </div>
+
+    <div class="frame-toast-manager">
+      <div class="frame-toast" id='sampleToast'>Message sent<button onclick='this.parentElement.classList.remove("show"); document.getElementById("toastButton").innerHTML = "Show Toast"' type="button" class="frame-toast-close"><span class="polaris-icon"><svg viewBox="0 0 20 20" focusable="false" aria-hidden="true"><path d="M11.414 10l6.293-6.293a.999.999 0 1 0-1.414-1.414L10 8.586 3.707 2.293a.999.999 0 1 0-1.414 1.414L8.586 10l-6.293 6.293a.999.999 0 1 0 1.414 1.414L10 11.414l6.293 6.293a.997.997 0 0 0 1.414 0 .999.999 0 0 0 0-1.414L11.414 10z" fill-rule="evenodd"></path></svg></span></button></div>
```
Comment (derekmorash) [inline demo/index.html]: The close button needs some kind of text to make it accessible to screen reader users. Using the `sr-only` class to visually hide the text but have it still be read by screen readers.
Markup could also use a little formatting.

```suggestion
      <div class="frame-toast" id='sampleToast'>Message sent
        <button onclick='this.parentElement.classList.remove("show"); document.getElementById("toastButton").innerHTML = "Show Toast"' type="button" class="frame-toast-close">
          <span class="sr-only">Close Toast</span>
          <span class="polaris-icon"><svg viewBox="0 0 20 20" focusable="false" aria-hidden="true"><path d="M11.414 10l6.293-6.293a.999.999 0 1 0-1.414-1.414L10 8.586 3.707 2.293a.999.999 0 1 0-1.414 1.414L8.586 10l-6.293 6.293a.999.999 0 1 0 1.414 1.414L10 11.414l6.293 6.293a.997.997 0 0 0 1.414 0 .999.999 0 0 0 0-1.414L11.414 10z" fill-rule="evenodd"></path></svg></span>
        </button>
      </div>
```

**Comment by derekmorash** [comment]
Hey thanks for the PR! This is great!

The reason this component was left out was because at Able Sense we build only embedded apps, so we have access to the Embedded app SDK and also the new APP Bridge that both have this feature.
https://shopify.dev/tools/embedded-app-sdk/methods#shopifyapp-flashnotice-message
https://shopify.dev/tools/app-bridge/actions/toast

But I have no problem adding this to the project. It's a relatively small amount of CSS so it's not a big deal.

I have one general comment about the CSS. It looks like you're using some CSS custom properties, I'm guessing these were just copied from the polaris docs. They do have fallbacks like `border-radius: var(--p-border-radius-wide, 3px);` where '--p-border-radius-wide' is the name of the property and '3px' is the fallback or default value. It might be a good idea to update these to use the SCSS variables inside bootstrap instead since this project doesn't use custom properties. For this one I'd make a SCSS variable called something like `$toast-border-radius` in the Alerts section of `_variables.scss`.

Also it might help to add your dummy JS to the demo/bootstrap-polaris.js file, just to keep markup clean. This file is for adding dummy/demo JS. I would add a unique id to the toast and the toast show button that you can use to target.

Other than that it looks great! If you have any questions just let me know. Once those things are addressed I'll gladly merge the changes.

Oh and to automatically have the changes in your repo added to the github pages demo you have to go to the settings of your forked repo and turn it on. Then you won't need to copy and paste it to your knagode.github.io repo!

**Comment by knagode** [comment]
Wow. Thanks for the hint regarding Github pages :) Awesome. 

We use this repo mostly because we want to build apps fast - it seems like we are faster without _heavy_ frameworks (Polaris). We also often build apps which also work outside Shopify scope and that is why we also don't like to stick to Polaris. This is also why we need simple Toast logic (we don't want to rely on Shopify too much) ..  

By using standard Bootstrap markup we are also able to use a lot of gems/scripts out of the box.

**Comment by derekmorash** [comment]
I completely understand where you're coming from, this is exactly why we built this project in the first place. We wanted more semantic and accessible markup and a framework that made it easier and faster to build with.

Thanks for the great work! I'd be happy to help out again with any other Polaris features you want to add in the future!

**Pull request merged**